### ISSUE TITLE
Custom command bar button is not counted when Narrator is on

### ISSUE THREAD

**Issue by yanbxice** [microsoft/fluentui#13937]
### Environment Information

- **Package version(s)**: all versions
- **Browser and OS versions**: Edge in Windows 10

### Describe the issue:

I want to customize one of the command bar buttons by using **onRender** attribute. It renders the button and the button works as expected. But when Narrator is on, the information is not announced correctly. e.g. I have 6 buttons on the command bar including one custom button. When focus goes to other buttons, it reads like "Menu item 1 of 5", "Menu item 2 of 5". The total number it reads out is 5 but there are actually 6 buttons. Also, Narrator doesn't announce "Menu item x of x" when focus on it.

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/yanbxice/pen/ExPEpav?editable=true%3Dhttps%3A%2F%2Ffabricweb.z5.web.core.windows.net%2Foufr%2F7.121.4%2F

#### Actual behavior:

Custom button is not counted as one of the command bar buttons

#### Expected behavior:

Custom button should be counted as one of the command bar buttons. In the above example, the total number of buttons should be 6 and when focus is on the custom button, it should also reads "Menu item x of x"

**Comment by paulgildea** [comment]
@joschect - For custom CommandBar buttons using `onRender` is there any specific work an author needs to do to ensure that it's included the a11y tree?

**Comment by yanbxice** [comment]
Tried with role='menuitem' on button and resolved the problem.
